file_templates/stack_license_f90/example.f90

```diff
@@ -1,6 +1,6 @@
 ! 
 ! LSST Data Management System
-! Copyright 2008-2017  AURA/LSST.
+! Copyright 2018 Association of Universities for Research in Astronomy
```

Comment:
Will the comparison in the travis file break on Jan 1, 2019?

Reply:
I'm not sure there's a way around that, but I'm not sure updating all the examples would be how I'd want to spend my new year's day.